app/bundles/EmailBundle/Helper/MessageHelper.php

```diff
@@ -1499,6 +1530,17 @@ public static function parseDsn($dsn_msg, $dsn_report, $debug_mode = false)
                         $result['rule_cat'] = 'command_reject';
                         $result['rule_no']  = '0225';
                     }
+					
+					/*
+					 * rule: mailbox restricted;
+					 * sample:
+					 * The error that the other server returned was:
+					 * Diagnostic-Code: SMTP; 550 5.7.1 RESOLVER.RST.NotAuthorized; not authorized
+					 */
+					elseif (preg_match('/NotAuthorized/is', $diag_code)) {
+						$result['rule_cat'] = 'command_reject';
+						$result['rule_no']  = '0225';
+					}
```

Comment:
There is some additional indentation. Try to use the PHP CS Fixer triggered by pre-commit hook: https://github.com/mautic/mautic/blob/staging/.github/CONTRIBUTING.md#coding-standards . It will take care of formatting for you.